packages/shared/src/server/services/S3StorageService.ts

```diff
@@ -121,17 +121,20 @@ export class S3StorageService {
     }
   }
 
-  private async getSignedUrl(
+  public async getSignedUrl(
     fileName: string,
     ttlSeconds: number,
+    asAttachment: boolean = true,
   ): Promise<string> {
     try {
       return await getSignedUrl(
         this.client,
         new GetObjectCommand({
           Bucket: this.bucketName,
           Key: fileName,
-          ResponseContentDisposition: `attachment; filename="${fileName}"`,
+          ResponseContentDisposition: asAttachment
```

Comment:
why did you need to change this?

Reply:
Otherwise file is always downloaded as attachment (as we have with batch exports). For media we want to render inline.